### DISCUSSION TITLE
[Request] Shift Click start, Shift Click End, than Add Tag to ALL selected images.

### DISCUSSION THREAD

**Discussion by Manniala** [zanllp/infinite-image-browsing#442] [Ideas]
Hey, thx for a great Extension.

Got a request, would it be possible to add the feature where you Shift Click a start and end, or Ctrl Click Images, than place Tag on ALL selected at once, either with the hotkeys them self, like you can if you open the images and the Prompt info... 

Some times i batch make a lot of image, before i go and tag them, and it can take a little while to tag and delete the images, but deleting can be done by method mentioned, just not the Tagging.

**Comment by Manniala** [comment]
Saw it was under Issues ( did not see it there, since it should had been under Discussions :D )

**Comment by zanllp** [comment]
Already supported.
https://github.com/zanllp/sd-webui-infinite-image-browsing/wiki/Change-log
